tests/test_logging.py

```diff
@@ -26,25 +26,47 @@ def test_setOverrideLogLevel(self):
         self.assertEqual(notherlogger.getEffectiveLevel(), logging.ERROR)
         setOverrideLogLevel('INFO')
         somelogger = getLogger('foo.bar')
-        setOverrideLogLevel('INFO')
-
-
-class TestLogging2(TestCase):
-
-    def setUp(self):
-        initLogging()
 
     def test_getLevelName(self):
         self.assertEqual(getLevelName('ERROR'), logging.ERROR)
         self.assertEqual(getLevelName('FATAL'), logging.ERROR)
         self.assertEqual(getLevelName('OFF'), logging.CRITICAL)
+        setOverrideLogLevel('INFO')
+
+
+class TestLoggingConfiguration(TestCase):
 
-    def test_configfile(self):
+    def test_tmpConfigfile(self):
+        self.assertNotEqual(logging.getLogger('').getEffectiveLevel(), logging.NOTSET)
```

Comment:
I don't get why it is `NOTSET` now. The module-level import-time `initLogging` (or any of `TestLogging`'s changes) _should_ still be in effect, right?

Reply:
Can you please add the first part of my above proposal to avoid host-side interference _already_, namely adding `HOME=$(CWD) ` to the pytest call in the makefile? (Just to be on the safe side that no one's flying ocrd_logging.conf files get in the way.)

Reply:
What's `$(CWD)`? You do not mean `$(PWD)`?

Reply:
> What's `$(CWD)`? You do not mean `$(PWD)`?

CWD is make's current working directory (i.e. the one it chdired to and actually ran from, including `-C` indirection and all). This is the safest bet.

PWD is a shell variable, and would in general be made available under that name within make, too. But it is not much use, because it only reflects the CWD at the place `make` was started from (without chdir since then).

Reply:
$(CWD) has not value set when I run `make test`on my Ubuntu 18.04 LTS ... ?

Reply:
> I don't get why it is `NOTSET` now. The module-level import-time `initLogging` (or any of `TestLogging`'s changes) _should_ still be in effect, right?

The assertion is, that it is NOT logging.NOTSET so it must be set to something. 
In several local test runs it was almost everytime `logging.DEBUG` and one time `logging.INFO`. I don't why, so I handled it rather pragmatic.

Reply:
> $(CWD) has not value set when I run `make test`on my Ubuntu 18.04 LTS ... ?

Are you sure? Did you write `HOME=$(CWD) python -m pytest` etc in the recipe (and then checked for the value of `HOME` in Python)?

Reply:
No, I just echo' d `$(CWD)` straight from the Makefile. It was empty, whereas `$(PYTHON)` yields `python`.
I'm no expert regarding make-stuff, therefore the probably absurd question.

Reply:
So sorry! I meant `$(CURDIR)`. A memory glitch...